staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/testing/testserver.go

```diff
@@ -58,6 +59,8 @@ type Logger interface {
 	Errorf(format string, args ...interface{})
 	Fatalf(format string, args ...interface{})
 	Logf(format string, args ...interface{})
+	Log(args ...interface{})
+	Helper()
```

Comment:
These new methods are needed for `ktesting`.

This is a staging repo, therefore I decided to not drop the interface completely - there might be downstream consumers which depend on it.